meilisearch-http/tests/common.rs

```diff
@@ -0,0 +1,146 @@
+#![allow(dead_code)]
```

Comment:
Please remove this.

Reply:
also on test ? It add warning on this files for each functions